app/models/profile_validation_decision.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class ProfileValidationDecision < ApplicationRecord
+  has_paper_trail
+
+  validates :note, presence: true
```

Comment:
Is the note required?

Reply:
Good question - I think best way to find out is to make it required and wait for feedback :P